src/figuresClasses.ts

```diff
@@ -1,19 +1,78 @@
 export interface Figure {
+  shape: string;
+  color: string;
```

Comment:
Describe all shapes and colors used in this task.

Reply:
why?